### DISCUSSION TITLE
Fractional scaling with Regolith 3.0 Beta

### DISCUSSION THREAD

**Discussion by m-wachter** [regolith-linux/regolith-desktop#892] [Q&A] [answered]
Hi,

first of all thanks for this great project.

I am just testing Regolith 3.x on wayland together with Ubuntu 23.04 on a laptop with a 2256x1504 (3:2) resolution display. When using the "normal" Ubuntu-session I am able to select a fractional (150%) scaling value in the display settings dialog, but with Regolith 3.x I am only able to select 100% or 200%

How can I configure the fractional scaling?

Greetings,

Michael.

**Comment by HeyTimApple** [comment]
I thought I could do this with the way it's done in Gnome Wayland...which I did and it works.. 
The Gnome way is
`gsettings set org.gnome.mutter experimental-features "['scale-monitor-framebuffer']"`

Again, this isn't working in Regolith-Settings, which does make sense. I wonder if there is a way to make it happen.

**Comment by SoumyaRanjanPatnaik** [comment, marked as answer]
Hey Michael. Which beta are you using? Fractional scaling was introduced in regolith 3.0 Beta 4,

**Reply by m-wachter** [reply]
I upgraded my packages to the current version and now I can select the 150% setting.

Thank you very much. I am looking forward to the final release.

Greetings,

Michael.
